matrix-sdk-android/src/rustCrypto/java/org/matrix/android/sdk/internal/crypto/RustCryptoService.kt

```diff
@@ -360,9 +360,7 @@ internal class RustCryptoService @Inject constructor(
     }
 
     override fun getLiveCryptoDeviceInfo(userIds: List<String>): LiveData<List<CryptoDeviceInfo>> {
-        return olmMachine.getLiveDevices(listOf(myUserId)).map {
-            it.filter { it.userId == myUserId }
-        }
+        return olmMachine.getLiveDevices(userIds)
```

Comment:
Good catch. The mapping to filter with `myUserId` was also quite useless, no? Anyway it's gone now :)